modules/Ecommerce/EcoBrand/Requests/Dashboard/UpdateEcoBrandDashboardRequest.php

```diff
@@ -44,6 +47,9 @@ public function messages(): array
             'description.en.string' => __('ecobrand::validation.description_en_string'),
             'description.en.max' => __('ecobrand::validation.description_en_max'),
             
+            // Status validation messages
+            'is_active.boolean' => 'حالة التفعيل يجب أن تكون صحيح أو خطأ',
```

Comment:
The validation message should be using the translation helper `__()` like other messages in this file for consistency.

```suggestion
            'is_active.boolean' => __('ecobrand::validation.is_active_boolean'),
```